File: server/match/src/main/java/info/chenliang/moba/message/BattleToMatchDispatcher.java
package info.chenliang.moba.message;
import java.io.*;
import info.chenliang.talky.*;
public class BattleToMatchDispatcher {
public static void dispatch(DataInputStream dis, BattleToMatchProxy proxy) throws Exception{
int fid = (int)(dis.read()&0xFFFF);
switch(fid)
{
}
}
}

File: client/Assets/Scripts/Message/info/chenliang/moba/message/BattleToMatchDispatcher.cs
namespace info.chenliang.moba.message{
using System;
using System.IO;
using MiscUtil;
using MiscUtil.IO;
using MiscUtil.Conversion;
using Talky;
public class BattleToMatchDispatcher {
public static void Dispatch(MemoryEndianBinaryReader reader, BattleToMatchProxy proxy) {
int fid = reader.ReadByte();
switch(fid)
{
}
}
}
}
